SwrveSDK/SDK/Talk/SwrveMessageController.m

```diff
@@ -790,36 +784,39 @@ -(SwrveMessage*)getMessageForEvent:(NSString *)event
         
         // If QA enabled, send message selection information
         if(self.qaUser != nil) {
-            [self.qaUser trigger:event withMessage:result withReason:campaignReasons withMessages:campaignMessages];
+            [self.qaUser trigger:eventName withMessage:result withReason:campaignReasons withMessages:campaignMessages];
```

Comment:
if we're triggering an event with a payload. How difficult would it be to add it as part of the display / hasShown event?